src/components/IncomingMessageList.jsx

```diff
@@ -263,6 +288,9 @@ const mapQueriesToProps = ({ ownProps }) => ({
                 text
                 isFromContact
               }
+              optOut {
+                cell
```

Comment:
future cache-merge note: this should be `id` (which will actually be fake) -- if this particular view needs the number, then it can get it from message.contactNumber.  That will be more cache friendly